rust/lanzatool/src/pe.rs

```diff
@@ -167,17 +167,17 @@ fn write_to_tmp(
     Ok(path)
 }
 
-fn esp_relative_path_string(esp: &Path, path: &Path) -> String {
+fn esp_relative_path_string(esp: &Path, path: &Path) -> Result<String> {
```

Comment:
This is a good suggestion. Since these changes were pulled out into a (now already merged) separate PR (#42), I have opened another PR which implements your requested changes: #43